fe/fe-core/src/main/java/org/apache/doris/alter/Alter.java

```diff
@@ -1193,8 +1220,8 @@ private void sortReplicasByBackendCount(List<Replica> replicas,
     }
 
     private void markReplicasForDropping(List<Replica> replicas, int replicasToDrop,
-                                  Map<Long, Long> tableBeToReplicaNumMap,
-                                  Map<Long, Long> partitionBeToReplicaNumMap) {
+            Map<Long, Long> tableBeToReplicaNumMap,
```

Comment:
You need to change your IDEA's reformat logic to only reformat the line you changed